src/pages/design/icons/chrome-icons.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Chrome Icons
```

Comment:
Same thing here, sentence case. Lowercase i for icons

Also, tangent: Icon overview and and icon library page titles aren't in sentence case either. Mind if you could catch those as well, Deric? Thank you!!

Reply:
Sounds good! Thanks for the review!